src/test_workflow/integ_test/distribution_deb.py

```diff
@@ -30,16 +30,14 @@ def install(self, bundle_name: str) -> None:
         logging.info("deb installation requires sudo, script will exit if current user does not have sudo access")
         deb_install_cmd = " ".join(
             [
-                'sudo',
-                'env',
-                'OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!',
-                '&&',
                 'sudo',
                 'dpkg',
                 '--purge',
                 self.filename,
                 '&&',
                 'sudo',
+                'env',
+                'OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!',
```

Comment:
What about versions less than 2.12.0? Maybe use `get_password` here similar to https://github.com/opensearch-project/opensearch-build/pull/4334/files#diff-68a045772f64ada86fcfd23d4e6685b142ed16c1c92ff159601592797e8d4bceR90

Reply:
@gaiksaya versions less than 2.12.0 will not get affected at all.
Because they dont even care about this env var.